flake.nix

```diff
@@ -24,6 +24,7 @@
       dell-inspiron-5509 = import ./dell/inspiron/5509;
       dell-inspiron-5515 = import ./dell/inspiron/5515;
       dell-inspiron-7405 = import ./dell/inspiron/7405;
+      dell-latitude-3340 = import ./dell/latitude/3340;
```

Comment:
Please also import the profile in your flake.nix.

Reply:
sorry, uncertain what this means. there are no imports in this `flake.nix`.

Reply:
Ah. sorry. I mean the README maybe.

Reply:
I actually no longer remember what I was on about here.